docs/_includes/chart.html

```diff
@@ -0,0 +1,602 @@
+<div id="chart">
+    {% include section-header.html name="chart" version=page.versions.chart %}
+    <p>The chart module facilitates the layout of a chart, chart-legend, and any other applicable text (e.g. metric or subtitle text).</p>
+    <h3>Stacked chart with 3 segments</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div aria-label="Donut chart title" class="chart chart--stacked">
```

Comment:
Do we need `chart--stacked`? For the default/base case we should typically have no modifiers. For variations we start adding modifiers.

Reply:
I was trying to avoid overwriting any properties from the base chart class, so chart--stacked was needed for a couple of additional properties.  I could move them to the base chart class but will have to override a couple properties for the other layouts

Reply:
for more clarity the chart--stacked is setting a max width, which would bi overwritten for the chart--composition and chart--horizontal version.  Let me know if you would prefer I remove the --stacked class and just override that max-width